private/buf/cmd/buf/command/beta/registry/webhook/webhookdelete/webhookdelete.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webhookdelete
+
+import (
+	"context"
+
+	"github.com/bufbuild/buf/private/buf/bufcli"
+	"github.com/bufbuild/buf/private/pkg/app/appcmd"
+	"github.com/bufbuild/buf/private/pkg/app/appflag"
+	"github.com/bufbuild/buf/private/pkg/command"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+const (
+	webhookIDFlagName = "webhook_id"
+)
+
+// NewCommand returns a new Command
+func NewCommand(
+	name string,
+	builder appflag.Builder,
+) *appcmd.Command {
+	flags := newFlags()
+	return &appcmd.Command{
+		Use:   name,
+		Short: "Delete a repository webhook with a webhook id.",
+		Args:  cobra.ExactArgs(0),
+		Run: builder.NewRunFunc(
+			func(ctx context.Context, container appflag.Container) error {
+				return run(ctx, container, flags)
+			},
+			bufcli.NewErrorInterceptor(),
+		),
+		BindFlags: flags.Bind,
+	}
+}
+
+type flags struct {
+	WebhookID string
+}
+
+func newFlags() *flags {
+	return &flags{}
+}
+
+func (f *flags) Bind(flagSet *pflag.FlagSet) {
+	flagSet.StringVar(
+		&f.WebhookID,
+		webhookIDFlagName,
+		"",
+		"The webhook ID to delete.",
+	)
+	_ = cobra.MarkFlagRequired(flagSet, webhookIDFlagName)
+}
+
+func run(
+	ctx context.Context,
+	container appflag.Container,
+	flags *flags,
+) error {
+	bufcli.WarnBetaCommand(ctx, container)
+	input, err := bufcli.GetInputValue(container, "", ".")
+	if err != nil {
+		return err
+	}
+	storageosProvider := bufcli.NewStorageosProvider(false)
+	runner := command.NewRunner()
+	_, moduleIdentity, err := bufcli.ReadModuleWithWorkspacesDisabled(
```

Comment:
This doesn't seem right - you're reading an input from the first container, which as defined in `bufcli.GetInputValue`, is the first argument, however this command takes zero arguments. You need to specify a way to pass the remote, such as `buf beta registry webhook delete --remote buf.build --id ID`

Reply:
Yep! I was hoping to add this ability after this PR since that would be analogous for the 3 functionalities we wanted

Reply:
I think I'm more saying the above is broken :-)

Reply:
I think I addressed what you were suggesting: I added a required remote flag (`--remote`) for the user to pass down the remote for the webhook cli commands

Reply:
looks good